components/Article/ArticlePage.js

```diff
@@ -0,0 +1,89 @@
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                    <div className="article-content">
+                        <p className="article-content-header">What Should be in a First Aid kit?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                </div>
+                <div className="content-wrapper">
+                    <div className="article-content article1">
+                        <p className="article-content-header">Why First Aid?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                    <div className="article-content">
+                        <p className="article-content-header">What Should be in a First Aid kit?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                </div>
+                <div className="content-wrapper">
+                    <div className="article-content article1">
+                        <p className="article-content-header">Why First Aid?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                    <div className="article-content">
+                        <p className="article-content-header">What Should be in a First Aid kit?</p>
+                        <p className="article-read">
+                            {ARTICLE_READ}
+                            <a className="article-content-a" href="/articlePage">
+                                {READ_MORE}
+                                ...
+                            </a>
+                        </p>
+                    </div>
+                </div>
```

Comment:
all of this content can created through an array mapping

Reply:
Will do that once I start connecting it to the api directly